package/XorShift32.roc

```diff
@@ -27,11 +28,4 @@ u32 = embed \@XorShift32 state ->
     |> \s -> Num.bitwiseXor s (Num.shiftLeftBy s 5)
     |> \s -> (@XorShift32 s, s)
 
-u64 =
-    x <- u32 |> andThen
-    y <- u32 |> andThen
-
-    (Num.toU64 x)
-    |> Num.shiftLeftBy 32
-    |> Num.bitwiseOr (Num.toU64 y)
-    |> return
+u64 = u64viau32
```

Comment:
The error is the same for

```
u64 = embed \state ->
    state
    |> Generator.run u64viau32
```

Reply:
type annotation fixed that but the result is the same as for https://github.com/wontem/roc-rand/pull/3

```
u64 : Generator XorShift32 U64
u64 = embed \state ->
    state
    |> Generator.run u64viau32
```